src/store/index.js

```diff
@@ -23,7 +24,9 @@ export default new Vuex.Store({
     },
     async find({ state }, query = {}) {
       const collection = state.connection.db.collection(state.collectionName)
-      state.records = await collection.find(query).toArray()
+      const cursor = await collection.find(query)
+      state.totalNumberOfResults = await cursor.count()
```

Comment:
would you consider not `await`ing count? i think it would result in faster query times. it's ok if we fetch this async.

Reply:
I've changed method call to synchronous mode and also changed the display decision based on the value of `totalNumberOfResults`.